server/core/terraform/async_client_test.go

```diff
@@ -150,7 +150,7 @@ func TestDefaultClient_RunCommandAsync_ExitOne(t *testing.T) {
 	outCh := client.RunCommandAsync(ctx, prjCtx, path, args, map[string]string{}, nil, workspace)
 
 	out, err := waitCh(outCh)
-	ErrEquals(t, fmt.Sprintf(`running "/bin/sh -c echo dying && exit 1" in %q: exit status 1`, path), err)
+	ErrEquals(t, fmt.Sprintf(`running "/usr/bin/sh -c echo dying && exit 1" in %q: exit status 1`, path), err)
```

Comment:
why did these change?

Reply:
the location of `sh` in `$PATH` changed from the old container.

are these run locally as well? ideally the assertion would ignore the path of `sh`.

Reply:
yeah its probably fine, we can fix it later.